Module/randomize.py

```diff
@@ -55,8 +54,9 @@ def populateLocations(self, excludeWorlds):
 
         self._validLocationListDonald = [location for location in self._allLocationListDonald if not set(location.LocationTypes).intersection(excludeWorlds)]
 
-    def populateItems(self, promiseCharm = False, startingInventory=[]):
-        validItemList = Items.getItemList() + Items.getSupportAbilityList() + Items.getActionAbilityList()
+    def populateItems(self, promiseCharm = False, startingInventory=[], abilityListModifier=lambda l: l):
+        validItemList = Items.getItemList() + abilityListModifier(Items.getSupportAbilityList() + Items.getActionAbilityList())
```

Comment:
Results in a `TypeError` when option is not selected (NoneType is not callable).

Reply:
fixed